src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -212,6 +213,10 @@ class ClusterMonitorActor(val cluster: Cluster,
     } yield res
   }
 
+  private def getRegion: Option[String] = {
+    cluster.labels.get(dataprocConfig.terraRegionLabel.get) flatMap WorkbenchProjectLocation.fromName map(_.storageLocation)
```

Comment:
it's probably all the same in the end but we should probably tweak `WorkbenchProjectLocation` so we can easily use a `computeRegion` instead of `storageLocation`

Reply:
you should be able to call computeRegionAndZones which gives you a map of regions -> zones.  It's a map because I could see in regions where we use a multi-region bucket we might want to support multiple regions.  In this specific case, I would just take the first one since (a) we don't have multiple in the hackathon and (b) even if we did, we don't have anything more intelligent to do at this point!

Reply:
...and they are different b/c for multi-region buckets, storage location isn't the same as a compute region (e.g. 'us' vs 'us-central1')